client.go

```diff
@@ -15,12 +15,12 @@ import (
 	"golang.org/x/crypto/ssh"
 )
 
-// InternalInconsistency indicates the packets sent and the data queued to be
+// ErrInternalInconsistency indicates the packets sent and the data queued to be
 // written to the file don't match up. It is an unusual error and usually is
 // caused by bad behavior server side or connection issues. The error is
 // limited in scope to the call where it happened, the client object is still
 // OK to use as long as the connection is still open.
-var InternalInconsistency = errors.New("internal inconsistency")
+var ErrInternalInconsistency = errors.New("internal inconsistency")
```

Comment:
This is public, so it can't be changed without breaking backwards compatibility. It either needs to be kept as is or you could make the old name an alias for the new.